core/src/test/java/org/infinispan/functional/distribution/rehash/FunctionalTxTest.java

```diff
@@ -187,14 +188,15 @@ public BlockingStateConsumer(StateConsumer delegate) {
       }
 
       @Override
-      public void applyState(Address sender, int topologyId, boolean pushTransfer, Collection<StateChunk> stateChunks) {
+      public CompletionStage<?> applyState(Address sender, int topologyId, boolean pushTransfer, Collection<StateChunk> stateChunks) {
          expectLatch.countDown();
          try {
             assertTrue(blockLatch.await(10, TimeUnit.SECONDS));
          } catch (InterruptedException e) {
             throw new RuntimeException(e);
          }
          super.applyState(sender, topologyId, pushTransfer, stateChunks);
+         return null;
```

Comment:
Should this return the value of the previous line?

Reply:
Good catch